libs/MVS/SceneDensify.cpp

```diff
@@ -1735,6 +1737,8 @@ void Scene::DenseReconstructionEstimate(void* pData)
 			}
 			// try to load already compute depth-map for this image
 			if (depthData.Load(ComposeDepthFilePath(idx, "dmap"))) {
+				if(!exportDmapOnly && access(ComposeDepthFilePath(idx, "pfm").c_str(),0) != -1)
+					depthData.depthMap.Load(ComposeDepthFilePath(idx, "pfm"));	
```

Comment:
use a temporary variable for ComposeDepthFilePath(idx, "pfm") to avoid composing path twice